src/components/pseudonym/repo/pseudonym.repo.ts

```diff
@@ -0,0 +1,30 @@
+import type { ObjectId } from '../../../../types';
+
+import { PseudonymModel, Pseudonym } from './db/pseudonym';
+import { DeleteManyResult, deleteManyResult } from '../../helper/repo.helper';
+import { validateObjectId } from '../../helper/validation.helper';
+
+const byUserFilter = (userId: ObjectId): { userId: ObjectId } => ({ userId });
+
+/**
+ * Return pseudonyms for userId
+ * @param userId
+ */
+const getPseudonymsForUser = async (userId: ObjectId): Promise<Pseudonym[]> => {
+	validateObjectId({ userId });
```

Comment:
Do we still need this?

Reply:
Ok. We need the check because ts does only compile time validation, and this function checks it on run time. :)